Depth-Anything-V2-list3/depth_anything_v2/util/blocks.py

```diff
@@ -120,28 +123,26 @@ def __init__(
 
         self.size=size
 
-    def forward(self, *xs, size=None):
+    def forward(self, x1: torch.Tensor, x2: Optional[torch.Tensor] = None, size: Optional[List[int]] = None):
```

Comment:
What is `x1` and `x2`?

Reply:
I'm changing the names to `xs0` and `xs1`, and adding documentation.